gprofiler/profilers/python.py

```diff
@@ -243,7 +251,8 @@ def _profile_process(self, process: Process) -> ProfileData:
 
     def _select_processes_to_profile(self) -> List[Process]:
```

Comment:
Instead of modifying this function, add `if is_windows()` and call your new function in that case, otherwise fall-through the old behavior

Reply:
Done